_integration-schemas/shiphero/v1/orders.md

```diff
@@ -0,0 +1,445 @@
+---
+tap: "shiphero"
+version: "1.0"
+
+name: "orders"
+doc-link: "https://shipheropublic.docs.apiary.io/#reference/orders/get-orders/get-orders"
+singer-schema: "https://github.com/singer-io/tap-shiphero/blob/master/tap_shiphero/schemas/orders.json"
+description: |
+  The `{{ table.name }}` table contains info about the orders in your {{ integration.display_name }} account.
+
+replication-method: "Key-based Incremental"
+api-method:
+    name: "Get orders"
+    doc-link: "https://shipheropublic.docs.apiary.io/#reference/orders/get-orders/get-orders"
+
+replication-key:
+  name: "from"
```

Comment:
There is no `from` field on the record, but there is a date that we look at when we sync the `orders`.

Reply:
Right - this property shows the rep key as something that isn't an attribute on the record itself. Like a path parameter. I assumed it was called `from` after looking at the API docs, but if we're using something else, I'd like to note it so support understands what they're looking at. @luandy64

Reply:
I don't know what we should call it. This stream just runs until it is caught up with "today".

Reply:
Digging a bit deeper, these are the parameters I think: `updated_from` and `updated_to`

https://github.com/singer-io/tap-shiphero/blob/e25ea8a4b4b0b5d93ae33ef42ed1ce1ab544fa39/tap_shiphero/sync.py#L219-L220

Reply:
Hey, that works! Thank you!